CodeEdit/Features/CEWorkspace/Models/CEWorkspaceSettingsPage.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  CEWorkspaceSettingsPage.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 27/3/24.
+//
+
+import Foundation
+import SwiftUI
+
+/// A struct for a workspace settings page
```

Comment:
In general, please don't use things like "A structure for" as they don't provide any useful information